packages/credential-provider-node/src/index.ts

```diff
@@ -51,6 +54,7 @@ export function defaultProvider(
         : chain(
             fromEnv(),
             fromIni(init),
+            fromProcess(init),
```

Comment:
You also need to mock `fromProcess()` in each of the unit test case of `credential-provider-node`. like [here](https://github.com/aws/aws-sdk-js-v3/blob/master/packages/credential-provider-node/src/index.spec.ts#L344).

Reply:
Fixed this.  Add test for passing through config.